package.json

```diff
@@ -8,22 +8,38 @@
   "license": "Apache-2.0",
   "scripts": {
     "compile": "tsc",
-    "lint": "tslint --project .",
     "test": "ava",
     "clean": "gts clean",
-    "fix": "gts fix"
+    "cli": "node ./build/src/cli/main.js",
+    "lint": "eslint './src/**/*.ts' './test/**/*.ts'"
+  },
+  "dependencies": {
+    "@types/yargs": "^15.0.5",
+    "ava": "^3.10.0",
+    "yargs": "^15.4.1"
   },
   "devDependencies": {
     "@ava/typescript": "^1.1.1",
-    "@google/local-home-sdk": "^1.0.0",
+    "@google/local-home-sdk": "^1.1.0",
```

Comment:
Can we update this dependency in a separate pull request just to simplify things?

Reply:
This update is currently based on #39, so it includes all of its changes.  Everything past 383d230 should exclude changes common to #39.  Will rebase once #39 is merged.